node/src/accountant/mod.rs

```diff
@@ -2090,22 +2571,77 @@ mod tests {
         assert_eq!(
             report_accounts_payables_msg,
             &ReportAccountsPayable {
-                accounts: vec![account]
+                accounts: vec![account],
+                response_skeleton_opt: None,
             }
         );
         let notify_later_payables_params = notify_later_payables_params_arc.lock().unwrap();
         assert_eq!(
             *notify_later_payables_params,
             vec![
-                (ScanForPayables {}, Duration::from_millis(97)),
-                (ScanForPayables {}, Duration::from_millis(97)),
-                (ScanForPayables {}, Duration::from_millis(97))
+                (
+                    ScanForPayables {
+                        response_skeleton_opt: None
+                    },
+                    Duration::from_millis(97)
+                ),
+                (
+                    ScanForPayables {
+                        response_skeleton_opt: None
+                    },
+                    Duration::from_millis(97)
+                ),
+                (
+                    ScanForPayables {
+                        response_skeleton_opt: None
+                    },
+                    Duration::from_millis(97)
+                )
             ]
         )
     }
 
     #[test]
-    fn scan_for_payable_message_does_not_trigger_payment_for_balances_below_the_curve() {
+    fn start_message_triggers_no_scans_in_suppress_mode() {
+        init_test_logging();
+        let system = System::new("start_message_triggers_no_scans_in_suppress_mode");
+        let config = bc_from_ac_plus_earning_wallet(
+            AccountantConfig {
+                scan_intervals: ScanIntervals {
+                    payable_scan_interval: Duration::from_millis(1),
+                    receivable_scan_interval: Duration::from_millis(1),
+                    pending_payable_scan_interval: Duration::from_secs(100),
+                },
+                when_pending_too_long_sec: DEFAULT_PENDING_TOO_LONG_SEC,
+                suppress_initial_scans: true,
+                payment_thresholds: *DEFAULT_PAYMENT_THRESHOLDS,
+            },
+            make_wallet("hi"),
+        );
+        let payable_dao = PayableDaoMock::new(); // No payables: demanding one would cause a panic
+        let receivable_dao = ReceivableDaoMock::new(); // No delinquencies: demanding one would cause a panic
+        let peer_actors = peer_actors_builder().build();
+        let subject = AccountantBuilder::default()
+            .bootstrapper_config(config)
+            .payable_dao(payable_dao)
+            .receivable_dao(receivable_dao)
+            .build();
+        let subject_addr = subject.start();
+        let subject_subs = Accountant::make_subs_from(&subject_addr);
+        send_bind_message!(subject_subs, peer_actors);
+
+        send_start_message!(subject_subs);
+
+        System::current().stop();
+        system.run();
```

Comment:
Hopefully a panic in the actor would kill the system too... but if you have just one, it should work...